packages/tsconfig-reference/copy/en/options/exactOptionalPropertyTypes.md

```diff
@@ -16,7 +16,7 @@ interface UserDefaults {
 
 Without this flag enabled, there are three values which you can set `colorThemeOverride` to be: "dark", "light" and `undefined`.
 
-Setting the value to `undefined` will allow most JavaScript runtime checks for the existence to fail, which is effectively falsy. However, this isn't quite accurate `colorThemeOverride: undefined` is not the same as `colorThemeOverride` not being defined. For example `"colorThemeOverride" in settings` would have different behavior with `undefined` as the key compared to not being defined.
```

Comment:
The original text is the actual expression, like `"key" in obj`; I don't think this change is correct.

Reply:
Whoops, that didn't even dawn on me.

Mixing expressions in the middle of a sentence is a little difficult to understand though, as seen here, I assumed typos/formatting issues before the real intent. Moving that into the code block is an idea, but I'll revert.